### DISCUSSION TITLE
What is the purpose of librespot_player?

### DISCUSSION THREAD

**Discussion by Advik-B** [kokarare1212/librespot-python#252] [Q&A] [answered]
If I want to stream audio, should I use [librespot_player](https://github.com/kokarare1212/librespot-python/tree/main/librespot_player)?
or stick to the docs?

Also, WHERE ARE THE DOCS?

**Comment by kokarare1212** [comment, marked as answer]
librespot_player is ported code for [these](https://github.com/librespot-org/librespot-java/tree/dev/player), but it is incomplete (lacks playback features, Spotify Connect, etc.).
If you want to actually stream audio, you can use [this example](https://github.com/kokarare1212/librespot-python/blob/main/examples/player/main.py).

Documentation is not yet in place, but can be found at [this URL](http://librespot-python.rtfd.io/).

In fact, there are limits to what one person can maintain. haha
